desktop/conf.dist/hue.ini

```diff
@@ -194,6 +194,14 @@ http_500_debug_mode=false
 # ^\/.*$,^http:\/\/www.mydomain.com\/.*$
 ## redirect_whitelist=^(\/[a-zA-Z0-9]+.*|\/)$
 
+# Image version/ Build version of Hue
+# e.g.: hue_image_version="2022.2.2.1"
+## hue_image_version=
+
+# Name of the Hue host
+# eg.: selected_hue_vw="hue-hive-1"
+## selected_hue_vw=
```

Comment:
Nit: same `## selected_hue_vw=hue-hive-1`

Reply:
updated. Thanks for pointing it out.